doc/man1/openssl.pod

```diff
@@ -40,6 +40,9 @@ The B<openssl> program provides a rich variety of commands (I<command> in the
 SYNOPSIS above), each of which often has a wealth of options and arguments
 (I<command_opts> and I<command_args> in the SYNOPSIS).
 
+Detailed documentation and use cases for most standard subcommands is available
+(eg. L<x509(1)> or L<openssl-x509(1)>).
```

Comment:
It seems `eg.` should be `e.g.`

Reply:
I guess it should be "...are available..."

Reply:
"e.g.,", even (with comma)

Reply:
Right, absolutely